src/main/java/no/dcat/themes/service/CodesService.java

```diff
@@ -172,7 +173,9 @@ private SkosCode getLocationCode(String uri) {
 
     private List<SkosCode> readMediaTypeCodesFromFile() {
         try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("json/mediatypes.json")) {
-            return Arrays.asList(new ObjectMapper().readValue(inputStream, SkosCode[].class));
+            return Stream.of(new ObjectMapper().readValue(inputStream, SkosCode[].class))
+                .peek(skosCode -> skosCode.setUri(String.format("%s/%s", "https://www.iana.org/assignments/media-types", skosCode.getCode())))
```

Comment:
From javadoc: Peek exists solely to support debugging purposes. Use map/each instead.